helm/linode-cosi-driver/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for linode-cosi-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
```

Comment:
i know none of this works yet, but probably dont want this to be nginx?

Reply:
This is template generated by `helm create`, with manifests removed. I thought leaving the values will be a good starting point.